From e12f6456046a6711b9de2ad69fe3a77529a1a50a Mon Sep 17 00:00:00 2001
From: b07117 <b07117@freescale.com>
Date: Tue, 15 Nov 2011 13:08:54 +0800
Subject: [PATCH 209/635] ENGR00160808-2 console: remove the useless log

like: E/hwcomposer( 2163): invalid gralloc handle (at 0x10050)
It is introduced by new gralloc wrapper

Signed-off-by: xianzhong <b07117@freescale.com>
---
 mx6/hwcomposer/hwcomposer.cpp | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mx6/hwcomposer/hwcomposer.cpp b/mx6/hwcomposer/hwcomposer.cpp
index daa7b6d..8f8bee1 100755
--- a/mx6/hwcomposer/hwcomposer.cpp
+++ b/mx6/hwcomposer/hwcomposer.cpp
@@ -518,6 +518,9 @@ static int hwc_set(hwc_composer_device_t *dev,
     blit_device *bltdev = ctx->blit;
     for (size_t i=0 ; i<list->numHwLayers ; i++){
 		hwc_layer_t *layer = &list->hwLayers[i];
+            if(!layer->handle || ((private_handle_t *)layer->handle)->magic != private_handle_t::sMagic) {
+                continue;
+            }
 	    if (private_handle_t::validate(layer->handle) < 0) {
     		//HWCOMPOSER_LOG_INFO("2--it is not a valide buffer handle\n");
     		continue;
-- 
1.8.0

